Form in template (#9224)
demisto/content: .github/pull_request_template.md

<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->

## Contributing to XSOARs content?

> Make sure to register your contribution - [contribution registration](https://forms.gle/XDfxU4E61ZwEESSMA)  

Reviewer: ```suggestion
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)
```

> **The Pull Request review will only start after this form was filled**

Reviewer: ```suggestion
**The Pull Request will be reviewed only after the contribution registration form is filled.**
```
